### PR TITLE
KPV 2566 :: View logic to list bulletins and resend it

### PR DESCRIPTION
**Pull request by pverdu-kuorum**

### CODE REVIEW

File: grails-app/conf/UrlMappings.groovy [hunk 1]
```diff
@@ -425,6 +426,7 @@ class UrlMappings {
         name politicianMassMailingRemove: "/ajax/account/mass-mailing/$campaignId/remove"(controller: "newsletter", action: "removeCampaign")
         name politicianMassMailingTrackEvents: "/ajax/account/mass-mailing/$campaignId/trackEvents"(controller: "newsletter", action: "showTrackingMails")
         name politicianMassMailingTrackEventsResend: "/ajax/account/mass-mailing/$campaignId/trackEvents/resend/$tackingMailId"(controller: "newsletter", action: "resendEmail")
+        name politicianMassMailingBulletinResend: "/ajax/account/mass-mailing/$campaignId/resend/$contactId"(controller: "newsletter", action: "copyBulletinAndSend")
```
Comment (iduetxe) [inline grails-app/conf/UrlMappings.groovy]: Is the same as API. Is it a "resend"?

File: grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -2636,6 +2636,7 @@ tools.contact.edit.tabs.activity.info.clickRate.bigger = Click rate &\#62 {0}%
 tools.contact.edit.tabs.activity.info.clickRate.smaller = Click rate &\#60; {0}%
 tools.contact.edit.tabs.activity.openRate = <span>{0}</span>  Open rate
 tools.contact.edit.tabs.basic = Details
+tools.contact.edit.tabs.bulletins = Boletines
```
Comment (iduetxe) [inline grails-app/i18n/messages.properties]: This key is not in POE. You need to add it and merge DEV branch

File: grails-app/services/kuorum/util/rest/RestKuorumApiService.groovy [hunk 3]
```diff
@@ -266,6 +266,7 @@ class RestKuorumApiService {
         ACCOUNT_BULLETINS("/communication/campaign/bulletin/{userId}"),
         ACCOUNT_BULLETIN("/communication/campaign/bulletin/{userId}/{campaignId}"),
         ACCOUNT_BULLETIN_COPY("/communication/campaign/bulletin/{userId}/{campaignId}/copy"),
+        ACCOUNT_BULLETIN_COPY_AND_SEND("/communication/campaign/bulletin/{userId}/{campaignId}/copy/send/{contactId}"),
```
Comment (iduetxe) [inline grails-app/services/kuorum/util/rest/RestKuorumApiService.groovy]: Again, Could you think in a better name?

File: grails-app/views/contacts/editContact.gsp [hunk 4]
```diff
@@ -197,6 +199,10 @@
                     <formUtil:uploadContactFiles contact="${contact}"/>
                 </div>
             </div>
+
+            <div class="tab-pane" id="bulletins">
+                <modulesUtil:delayedModule mapping="politicianContactBulletin" params="${[contactId: contact.id]}" elementId="contact-bulletin-history-list-id"/>
```
Comment (iduetxe) [inline grails-app/views/contacts/editContact.gsp:204]: Uooo, you are a crack. How you find it?? Great!!

File: web-app/js/contacts.js [hunk 5]
```diff
@@ -458,6 +458,32 @@ $(function () {
         resend($link, $button);
     });
 
+    $('body').on('click','.resend-bulletin', function(e) {
+        e.preventDefault();
+        var $a = $(this)
+        var link = $a.attr("href")
+        $("#activity-resend-bulletin-confirm").modal("show");
```
Comment (iduetxe) [inline web-app/js/contacts.js:465]: Uooo!! WIth Modal!! Great

**Review by iduetxe — CHANGES_REQUESTED** [review]
Mainly, you need poe texts :)

File: grails-app/conf/UrlMappings.groovy (hunk 1 above)
Comment (pverdu-kuorum) [reply grails-app/conf/UrlMappings.groovy]: Changed to 

`politicianMassMailingBulletinCopyAndSend: "/ajax/account/mass-mailing/$campaignId/send/$contactId"(controller: "newsletter", action: "copyBulletinAndSend")`

File: grails-app/i18n/messages.properties (hunk 2 above)
Comment (pverdu-kuorum) [reply grails-app/i18n/messages.properties]: Added missed property

File: grails-app/services/kuorum/util/rest/RestKuorumApiService.groovy (hunk 3 above)
Comment (pverdu-kuorum) [reply grails-app/services/kuorum/util/rest/RestKuorumApiService.groovy]: Changed to:

.../{campaignId}/send/{contactId}

File: web-app/js/contacts.js (hunk 5 above)
Comment (pverdu-kuorum) [reply web-app/js/contacts.js:465]: It's a QA future request protection ;)

**Review by iduetxe — APPROVED** [review]

**Pull request merged**